lib/src/components/NxForm/stateful/__tests__/NxStatefulForm.test.tsx

```diff
@@ -441,12 +396,16 @@ describe('NxStatefulForm', function() {
     // only changing back to not having validationErrors counts
```

Comment:
does this test need to have a user.click event? Or is it considered addressed on line 365?

Reply:
See https://github.com/sonatype/sonatype-react-shared-components/pull/829/files/b444d99fb300a2f0c8894ee54a14abee26b45cbb#r1002072022 .  I add the click to each transition where the previous `showValidationErrors` value was `false`.

Reply:
> See https://github.com/sonatype/sonatype-react-shared-components/pull/829/files/b444d99fb300a2f0c8894ee54a14abee26b45cbb#r1002072022 . I add the click to each transition where the previous `showValidationErrors` value was `false`.

Ah I see, thanks for linking the explanation!